internal/pkg/cli/job_init.go

```diff
@@ -0,0 +1,297 @@
+	if err != nil {
+		e, ok := err.(*workspace.ErrFileExists)
+		if !ok {
+			return "", err
+		}
+		manifestExists = true
+		manifestPath = e.FileName
+	}
+	manifestPath, err = relPath(manifestPath)
+	if err != nil {
+		return "", err
+	}
+
+	manifestMsgFmt := "Wrote the manifest for job %s at %s\n"
+	if manifestExists {
+		manifestMsgFmt = "Manifest file for job %s already exists at %s, skipping writing it.\n"
+	}
+	log.Successf(manifestMsgFmt, color.HighlightUserInput(o.Name), color.HighlightResource(manifestPath))
+	log.Infoln(color.Help("Your manifest contains configurations like your container size and retry logic"))
+	log.Infoln()
+
+	return manifestPath, nil
+}
+
+func (o *initJobOpts) newManifest() (encoding.BinaryMarshaler, error) {
+	return &manifest.BackendService{}, nil
+}
+
+func (o *initJobOpts) askJobName() error {
+	if o.Name != "" {
+		return nil
+	}
+
+	name, err := o.prompt.Get(
+		fmt.Sprintf(fmtSvcInitSvcNamePrompt, color.Emphasize("name"), color.HighlightUserInput("scheduled job")),
+		fmt.Sprintf(fmtJobInitNameHelpPrompt, o.AppName()),
+		validateSvcName,
+		prompt.WithFinalMessage("Job name:"))
+	if err != nil {
+		return fmt.Errorf("get job name: %w", err)
+	}
+	o.Name = name
+	return nil
+}
+
+// askDockerfile prompts for the Dockerfile by looking at sub-directories with a Dockerfile.
+// If the user chooses to enter a custom path, then we prompt them for the path.
+func (o *initJobOpts) askDockerfile() error {
+	if o.DockerfilePath != "" {
+		return nil
+	}
+
+	// TODO https://github.com/aws/copilot-cli/issues/206
+	dockerfiles, err := listDockerfiles(o.fs, ".")
+	if err != nil {
+		return err
+	}
+
+	sel, err := o.prompt.SelectOne(
+		fmt.Sprintf(fmtSvcInitDockerfilePrompt, color.Emphasize("Dockerfile"), color.HighlightUserInput(o.Name)),
+		jobInitDockerfileHelpPrompt,
+		dockerfiles,
+		prompt.WithFinalMessage("Dockerfile:"),
+	)
+	if err != nil {
+		return fmt.Errorf("select Dockerfile: %w", err)
+	}
+
+	o.DockerfilePath = sel
+
+	return nil
+}
+
+// RecommendedActions returns follow-up actions the user can take after successfully executing the command.
+func (o *initJobOpts) RecommendedActions() []string {
+	return []string{
+		fmt.Sprintf("Update your manifest %s to change the defaults.", color.HighlightResource(o.manifestPath)),
+		fmt.Sprintf("Run %s to deploy your job to a %s environment.",
+			color.HighlightCode(fmt.Sprintf("copilot job deploy --name %s --env %s", o.Name, defaultEnvironmentName)),
+			defaultEnvironmentName),
+	}
+}
+
+// BuildJobInitCmd builds the command for creating a new service.
+func BuildJobInitCmd() *cobra.Command {
+	vars := initJobVars{
+		GlobalOpts: NewGlobalOpts(),
+	}
+	cmd := &cobra.Command{
+		Use:   "init",
+		Short: "Creates a new scheduled job in an application.",
+		Example: `
+  Create a "reaper" scheduled task to run once per day.
+  /code $ copilot job init --name reaper --dockerfile ./frontend/Dockerfile --rate "1 day"
```

Comment:
Maybe this? to be consistent with https://godoc.org/github.com/robfig/cron
```suggestion
  /code $ copilot job init --name reaper --dockerfile ./frontend/Dockerfile --rate "@daily"
```

Reply:
Task definitions let you specify either the `rate` or `cron` fields, which both have different syntax. I definitely want to be able to use the cron sugar but I think that rate (`3 minutes`), cron (`0 * * * * *`), and this special syntax (`@hourly`) are different enough that we can intelligently guess what customers mean if they type in any one of those three.

Reply:
Now that I think about it I want to ditch the `--rate` flag entirely for a more intelligent `--schedule` flag which can parse out cron, Go!cron, and rates via regex or a simple grammar.